src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -115,7 +115,7 @@ public function setId($id)
      *
      * @return StopwatchEvent The event
      */
-    public function startEvent($name, $category)
+    public function startEvent(string $name, ?string $category)
```

Comment:
looks like we need to update the parameter type-hint in lower branches then

Reply:
this seems still open to do

Reply:
#32793